powertools-parameters/src/main/java/software/amazon/lambda/powertools/parameters/exception/ProviderException.java

```diff
@@ -0,0 +1,12 @@
+package software.amazon.lambda.powertools.parameters.exception;
+
+public class ProviderException extends RuntimeException {
+
+    public ProviderException(Exception e) {
+        super(e);
+    }
+
+    public ProviderException(String message) {
```

Comment:
Guess this is unused ?

Reply:
removing ProviderException and throwing RuntimeException instead as [here](https://github.com/awslabs/aws-lambda-powertools-java/blob/master/powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/SqsUtils.java#L260)